docker/pdf-launch-scripts/mtc-moveit-launch.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+# Launch Docker container for ur-driver
+ROS_DISTRO=humble
+UR_TYPE="ur3e"
+ROBOT_IP=10.66.218.141
+REVERSE_IP=10.66.219.39
+
+# Turn off tool communication here? 
+docker run -it --network host --ipc=host --pid=host\
+    ghcr.io/nsls2/ur-hande-draft:latest \
+    /bin/bash -c ". /root/ws/install/setup.sh && \
+ros2 launch ur_hande_moveit_config mtc_moveit_rviz.launch.py \
```

Comment:
Future work: rename these files as we won't be using mtc.

Reply:
Deprecation notes in README. Added more [d15985b](https://github.com/maffettone/erobs/pull/15/commits/d15985bab41b72ca3a7d1a27db8d4381db7031ba)